src/config/constants/pools.ts

```diff
@@ -4,6 +4,8 @@ const pools: PoolConfig[] = [
   {
     sousId: 0,
     tokenName: 'CAKE',
+    tokenAddress: '0x0e09fabb73bd3ade0a17ecc321fd13a19e81ce82',
+    tokenImage: 'CAKE',
```

Comment:
We have two choices here: Just use the token name (lowercase) as the image name or specify the full image name. Since the token image will repeat (CAKE) let's use the token name, lowercase to derive the image name.

Reply:
@hachiojidev - Agree. We currently use `tokenName:` for the image mapping already. We don't need `tokenImage:`